ci/common/build.sh

```diff
@@ -7,27 +7,26 @@ build_make() {
 }
 
 build_deps() {
-  if [[ "${BUILD_32BIT}" == ON ]]; then
+  if test "x${BUILD_32BIT}" = xON ; then
     DEPS_CMAKE_FLAGS="${DEPS_CMAKE_FLAGS} ${CMAKE_FLAGS_32BIT}"
   fi
-  if [[ "${FUNCTIONALTEST}" == "functionaltest-lua" ]]; then
+  if test "x${FUNCTIONALTEST}" = "xfunctionaltest-lua" ; then
```

Comment:
@jamessan We have a different understanding of “builtin”. For me this is a shortcut to “built-in command” and `[[` definitely is not a builtin with this definition.